Tests/SentryTests/Networking/RateLimits/SentryEnvelopeRateLimitTests.m

```diff
@@ -0,0 +1,23 @@
+#import <XCTest/XCTest.h>
+#import "SentryEnvelopeRateLimit.h"
+#import "SentryTests-Bridging-Header.h"
+
+@interface SentryEnvelopeRateLimitTests : XCTestCase
+
+@end
+
+/**
+* This class exists only for testing SentryEnvelopeRateLimitTests with nil. All other tests are in
+* SentryEnvelopeRateLimitTests.swift
```

Comment:
l: is there a technical problem to not add to the very same class then?

Reply:
I didn't find a way how to pass `nil` in Swift to a method that doesn't allow `nil`. @HazAT do you know how to do this?

Reply:
good question, I don't think you can.

Reply:
I've had the same problem with Java/Kotlin and I've used reflection, tbh it's not worth it, I'd agree for this very specific cases to just create a ObjC class